src/operator/tensor/elemwise_binary_broadcast_op_basic.cc

```diff
@@ -120,8 +120,13 @@ Example::
    broadcast_mul(x, y) = [[ 0.,  0.,  0.],
                           [ 1.,  1.,  1.]]
 
```

Comment:
Shall we also edit https://github.com/apache/incubator-mxnet/blob/master/docs/api/python/ndarray/sparse.md#arithmetic-operations and add broadcast_mul / broadcast_div?

Reply:
Done